### ISSUE TITLE
Do you have a link to download the data set? Can you explain the training steps?

### ISSUE THREAD

**Issue by fangliangs** [zhaipro/unet#3]

**Comment by zhaipro** [comment]
图片是爬取来的，具体来源基本都忘了。mask是调用[美图](http://ai.meitu.com/doc/?id=16)获取的。
https://drive.google.com/drive/folders/1SFZgEaw0x3Z8F7_-SgVz5zs2-iTaTMPa?usp=sharing

**Comment by fangliangs** [comment]
ERR when checking input: expected input_1 to have 4 dimensions, but got array with shape (1, 1)

def data_generator(path, batch_size=2):
    '''data generator for fit_generator'''
    n = 3000
    i = 0
    idxs = np.arange(n)
    while True:
        x, y = [], []
        for b in range(batch_size):
            if i % n == 0:
                np.random.shuffle(idxs)
            fn = f'{path}/x/{idxs[i]}.jpg'
            im = cv2.imread(fn)
            x.append(im)
            fn = f'{path}/y/{idxs[i]}.jpg'
            im = cv2.imread(fn, 0)
            y.append(im)
            i = (i + 1) % n
        x = np.array(x, dtype='float32') / 255
        y = np.array(y, dtype='float32') / 255
        y.shape = y.shape + (1,)
        yield x, y
